xml/System.IO.Compression/CompressionLevel.xml

```diff
@@ -95,12 +95,12 @@
       </Docs>
     </Member>
     <Member MemberName="NoCompression">
-      <MemberSignature Language="C#" Value="NoCompression" />
-      <MemberSignature Language="ILAsm" Value=".field public static literal valuetype System.IO.Compression.CompressionLevel NoCompression = int32(2)" />
-      <MemberSignature Language="DocId" Value="F:System.IO.Compression.CompressionLevel.NoCompression" />
-      <MemberSignature Language="VB.NET" Value="NoCompression" />
-      <MemberSignature Language="C++ CLI" Value="NoCompression" />
-      <MemberSignature Language="F#" Value="NoCompression = 2" Usage="System.IO.Compression.CompressionLevel.NoCompression" />
+      <MemberSignature Language="C#" Value="NoCompression" FrameworkAlternate="xamarinmac-3.0" />
```

Comment:
these fields exist for several other monikers. Not sure why it's identifying as xamarin only.

Reply:
I've resolved this issue locally